drop/modules/aberrant-splicing-pipeline/Counting/03_filter_expression_FraseR.R

```diff
@@ -5,11 +5,14 @@
 #'  log:
 #'    - snakemake: '`sm str(tmp_dir / "AS" / "{dataset}" / "03_filter.Rds")`'
 #'  params:
-#'   - setup: '`sm cfg.AS.getWorkdir() + "/config.R"`'
 #'   - workingDir: '`sm cfg.getProcessedDataDir() + "/aberrant_splicing/datasets/"`'
 #'  input:
+#'   - setup: '`sm cfg.AS.getWorkdir() + "/config.R"`'
 #'   - theta:  '`sm cfg.getProcessedDataDir()+
 #'                  "/aberrant_splicing/datasets/savedObjects/raw-{dataset}/theta.h5"`'
+#'   - txdb: '`sm cfg.getProcessedDataDir() + "/aberrant_expression/{annotation}/txdb.db"`'
```

Comment:
Snakemake cannot find the txdb, because the `annotation` wildcard is not specified in the output.
I wouldn't annotate the fds here anyway, because it is not thought to be annotation-depedent. Rather, it would be more useful to only annotate in the steps after FRASER fitting